### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.4 - es. 3.17 (14 ottobre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#291] [Esercizi M-Z]
![11-10 - 14:30:20](https://github.com/user-attachments/assets/6ab7f14d-2e86-4eda-a56a-e1597f14627a)

**Comment by marcoseverini** [comment]
<img width="1250" alt="immagine" src="https://github.com/user-attachments/assets/e8f0cfb1-87ef-4c1a-b73e-1fa934fe5aea">

**Comment by alem1105** [comment]
<img width="861" alt="375965071-99ef3e10-d2d0-4a59-b5b7-1fa509a375a1" src="https://github.com/user-attachments/assets/08473161-8996-4726-986e-7903b85621e7">
